### PR TITLE
OLM metadata for 0.13.1

### PR DESCRIPTION
**Pull request by aliok**
Fixes #

## Proposed Changes

* OLM metadata for 0.13.1
*
*

**Release Note**

```release-note
NONE
```

### CODE REVIEW

**Comment by aliok** [comment]
/hold

Hold until https://github.com/operator-framework/community-operators/pull/1338 is merged

**Comment by k4leung4** [comment]
do we want to wait for 0.13.1?

it has this fix, https://github.com/knative/pkg/commit/122514b865f268704598561649e61bfb4bc41885

**Comment by aliok** [comment]
> do we want to wait for 0.13.1?

Yeah, let's wait until then.

**Comment by houshengbo** [comment]
@k4leung4 operator 0.13.0 has already included knative/pkg@122514b

File: deploy/olm-catalog/knative-serving-operator/0.13.0/knative-serving-operator.v0.13.0.clusterserviceversion.yaml [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+    capabilities: Basic Install
+    categories: Networking,Integration & Delivery,Cloud Provider,Developer Tools
+    certified: "false"
+    containerImage: gcr.io/knative-releases/knative.dev/serving-operator/cmd/manager:v0.12.2
```
Comment (k4leung4) [inline deploy/olm-catalog/knative-serving-operator/0.13.0/knative-serving-operator.v0.13.0.clusterserviceversion.yaml]: is this expected to be on 0.12.2?

**Comment by aliok** [comment]
/unhold 

https://github.com/operator-framework/community-operators/pull/1338 is merged

**Comment by aliok** [comment]
cc @houshengbo

**Comment by houshengbo** [comment]
/lgtm
@aliok PLZ be aware that we are targeting to release one operator for coming release-0.15.

**Pull request merged**